cloud/aws/terraform/common/s3notif/inputs.tf

```diff
@@ -25,8 +25,7 @@ resource "null_resource" "bucket-location-check" {
     command = <<EOT
 aws s3api get-bucket-location --bucket ${var.bucket_name} \
   | grep ${var.region} \
-  || echo 'Wrong region for source bucket' \
-  && exit 1
+  || (echo 'Wrong region for source bucket' && exit 1)
```

Comment:
since `echo` doesn't fail, you could probably just write `;` and don't let peope ponder about the conditional expression :-)